include/scan_class/scan_part_dyn.klc

```diff
@@ -152,18 +153,20 @@ ROUTINE log_pose
     origin = (ZEROPOS(1))
 
     --get position
-    p = `MOTION_DATA_FILE`__get_cart(rbt_pos)
+    -- p = `MOTION_DATA_FILE`__get_cart(rbt_pos)
+    $UFRAME = $MNUFRAME[CD_PAIR,USER_FRAME_NO]
+    $UTOOL = $MNUTOOL[CD_PAIR,TOOL_FRAME_NO]
```

Comment:
Should `pose__set_toolframe`, and `pose__set_userframe` be updated for CD_PAIR so you don't have to use the raw declarations?

Reply:
Actually, that was my mistake. The first arrg is for the group number and not for the CD_PAIR. So, I switch to using the pose function instead of the raw decleration.